src/gui_common/thrive_theme.tres

```diff
@@ -1,4 +1,4 @@
-[gd_resource type="Theme" load_steps=48 format=2]
```

Comment:
I'll revert the changes here as these seem a bit unsensical as to why Godot did these automatically.